x-pack/plugins/serverless_search/public/application/components/connectors_elastic_managed.tsx

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLink,
+  EuiPageTemplate,
+  EuiText,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React, { useMemo } from 'react';
+
+import { LEARN_MORE_LABEL } from '../../../common/i18n_string';
+import { PLUGIN_ID } from '../../../common';
+import { useConnectors } from '../hooks/api/use_connectors';
+import { useCreateConnector } from '../hooks/api/use_create_connector';
+import { useKibanaServices } from '../hooks/use_kibana';
+import { ElasticManagedConnectorCommingSoon } from './connectors/elastic_managed_connector_coming_soon';
+import { ConnectorsTable } from './connectors/connectors_table';
+
+import { CONNECTORS } from '../constants';
+
+export const ConnectorsElasticManaged = () => {
```

Comment:
Why is there a full connectors managed page here? We don't have any elastic managed connectors yet, and you're create connector creates a non-managed connector.

Reply:
This page aims for rendering the Coming soon page content for connectors as we do also with Web Crawlers. Consuming the <EuiPageTemplate> and rendering as children the      
 
```
<EuiPageTemplate.Section restrictWidth color="subdued">
        <ElasticManagedConnectorComingSoon />
 </EuiPageTemplate.Section>
```

We want to track how many users open this page so I guess this is the way to do it and get a dedicated URL route for this. But let me know if there is another way I should take care of.